pages/[lang]/index.tsx

```diff
@@ -117,8 +172,17 @@ export const Index: NextPage = () => {
       <SocialMediaSection />
 
       <style jsx>{`
-        .ways-to-join {
```

Comment:
Are the style changes needed for mixed group or is this more of a clean up. Asking because unfortunately I also rewrote some of this in my PR :(

Reply:
more of a clean up :) 
Maybe show me your changes and we can discuss it? I feel the way how was before wasn't very optimal